tar5/deep.py

```diff
@@ -1,11 +1,20 @@
 import glob
 
 
+
 def deep():
-    path = input("enter path:")
-    my_files = glob.glob(path + "\deep*")
-    for file in my_files:
-        print(file[len(path) + 1:])
+    """
+    print list of files called deep...

```

Comment:
First - Nice job improving your code.
But about the documentation - it's not exactly what the function does right? (It supposed to return a list of all the files that starts with "deep").